### PR TITLE
enhanced .htaccess files for supporting both Apache 2.2 and 2.4

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | #2938 
| Refs tickets      | -
| License           | MIT
| Changelog updated | no

### CODE REVIEW

File: src/config/.htaccess [hunk 1]
```diff
@@ -1,5 +1,27 @@
-deny from all
+# ------------------------------------------------------------------
+# Purpose of file: block any web access to unallowed files
+#                  stored under the /config directory
+# ------------------------------------------------------------------
+
+# Apache 2.2
+<IfModule !mod_authz_core.c>
+    Deny from all
```
Comment (cmfcmf) [inline src/config/.htaccess]: Shouldn't this also include `Order deny,allow`?

Comment (cmfcmf) [inline src/config/.htaccess]: I see, you removed `Order deny,allow` everywhere so this is a consistent change. I assume `Order deny,allow` is the default.

Comment (Guite) [inline src/config/.htaccess]: I basically didn't change the 2.2 rules at all, but only wrapped them into the `IfModule` check.

File: src/app/cache/.htaccess [hunk 2]
```diff
@@ -1,2 +1,14 @@
-Order deny,allow
```
Comment (cmfcmf) [inline src/app/cache/.htaccess]: You removed this one, that's why I noticed it.

Comment (Guite) [inline src/app/cache/.htaccess]: Good point! Will readd it immediately.

**Pull request merged**